### PR TITLE
fix grammar error in comment of type webhook configuration

### PR DESCRIPTION
**Pull request by zkyle95**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Make code easier to read

```release-note
None
```

### CODE REVIEW

File: staging/src/k8s.io/api/admissionregistration/v1/types.go [hunk 1]
```diff
@@ -114,7 +114,7 @@ const (
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// ValidatingWebhookConfiguration describes the configuration of and admission webhook that accept or reject and object without changing it.
+// ValidatingWebhookConfiguration describes the configuration of admission webhook that accept or reject object without changing it.
```
Comment (roycaihw) [inline staging/src/k8s.io/api/admissionregistration/v1/types.go:117]: I think both "and" should be "an". See https://github.com/kubernetes/kubernetes/commit/9ddea83a2ce0937cf0fc8f8c35614bb18e74cfad#diff-0c58bb8027e5eeb97b7d77d32c58a9def5816e61d141c4daf92bf351c707e91cR121

File: staging/src/k8s.io/api/admissionregistration/v1/types.go [hunk 2]
```diff
@@ -144,7 +144,7 @@ type ValidatingWebhookConfigurationList struct {
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// MutatingWebhookConfiguration describes the configuration of and admission webhook that accept or reject and may change the object.
+// MutatingWebhookConfiguration describes the configuration of admission webhook that accept or reject and may change the object.
```
Comment (roycaihw) [inline staging/src/k8s.io/api/admissionregistration/v1/types.go:147]: ditto

File: staging/src/k8s.io/api/admissionregistration/v1/types.go [hunk 3]
```diff
@@ -186,7 +186,7 @@ type ValidatingWebhook struct {
 	// Rules describes what operations on what resources/subresources the webhook cares about.
 	// The webhook cares about an operation if it matches _any_ Rule.
 	// However, in order to prevent ValidatingAdmissionWebhooks and MutatingAdmissionWebhooks
-	// from putting the cluster in a state which cannot be recovered from without completely
```
Comment (roycaihw) [inline staging/src/k8s.io/api/admissionregistration/v1/types.go:189]: I'm not the expert but the "from" seems correct to me.

**Review by roycaihw — COMMENTED** [review]
Please also run `make generated_files` to make the tests pass.

File: staging/src/k8s.io/api/admissionregistration/v1/types.go (hunk 1 above)
Comment (zkyle95) [reply staging/src/k8s.io/api/admissionregistration/v1/types.go:117]: Maybe "a list of" is more accurate as the field **Webhooks** is an array?

File: staging/src/k8s.io/api/admissionregistration/v1/types.go (hunk 3 above)
Comment (zkyle95) [reply staging/src/k8s.io/api/admissionregistration/v1/types.go:189]: Oops, my mistake, i'll restore it

File: staging/src/k8s.io/api/admissionregistration/v1/types.go (hunk 1 above)
Comment (roycaihw) [reply staging/src/k8s.io/api/admissionregistration/v1/types.go:117]: "a list of admission webhooks" sounds fine to me

**Comment by roycaihw** [comment]
Sorry for the late reply. You still need to run `make generated_files` to make the tests pass

**Pull request closed without merging**